src/test/resources/json/addfinaltransferconfirmation_data_error_invalid_consignmentid.json

```diff
@@ -2,7 +2,7 @@
   "data": null,
   "errors": [
     {
-      "message": "Invalid consignment id"
+      "message": "User '4ab14990-ed63-4615-8336-56fbb9960300' does not have access to consignment 'f67d1337-cbd0-4fd1-9eac-611489e7113f'"
     }
   ]
 }
```

Comment:
Missing line break

Reply:
I've been trying with the line breaks :smile: I've added this one and the one below.